src/components/RateAction/RateAction.jsx

```diff
@@ -109,12 +113,14 @@ RateAction.propTypes = {
     'reaction-sad',
     'reaction-okay',
   ]),
-  /** Adjusts the size of the component. One of 'lg', 'md', 'sm' */
-  size: PropTypes.oneOf(['lg', 'md', 'sm']),
+  /** Adjusts the size of the component. One of 'xl', 'lg', 'md', 'sm' */
+  size: PropTypes.oneOf(['xl', 'lg', 'md', 'sm']),
```

Comment:
Can you do me a quick favor and add the new size to the story? Thanks!

The 3 instances, from:

```
<RateAction
size={select('Size', { lg: 'lg', md: 'md', sm: 'sm' }, 'md')}
```

To:

```
<RateAction
size={select('Size', { xl: 'xl', lg: 'lg', md: 'md', sm: 'sm' }, 'md')}
```